tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,65 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def _add_host_label(instance):
+    """
+    Method add multiple labels to cephadmin node
+    Args:
+        instance: cluster instance
+    Returns:
+        1 -> Fail
+        0 -> Pass
+    """
+
+    node = instance.cluster.get_nodes()[0].hostname
+    cmd = (
+        "cephadm shell ceph orch host add {} node-exporter ".format(node)
+        + "crash osd installer grafana prometheus alertmanager"
+    )
+    try:
```

Comment:
I don't think we need to use try...catch block here. In case of failure `exec_command` will raise an exception.

Reply:
As suggested, Removed try block and added exception.